mars/services/core.py

```diff
@@ -19,6 +19,9 @@
 
 BandType = Tuple[str, str]
 
+class FilterType(enum.Enum):
```

Comment:
The name `FilterType` is not clear. Filter what?

Reply:
Currently nodes are filtered with a blocklist, thus calling this type of filter as one kind of `FilterType`. In other words, `FilterType` indicates different manners how nodes are filtered, even though there's only one till now.

Reply:
The notion of node filtering should be designed more carefully. Wherher the node is blacklisted or have some other labels (for instance, nodes having GPUs or other specified hardware) is parallel. We need to be careful when introducing global flags like this.

Reply:
And what's more, what is the position of this PR in the road map?